app/utilities/addAuthor.js

```diff
@@ -0,0 +1,8 @@
+function addAuthor(dataset, author) {
```

Comment:
With the modified `query` string I don't think we'll need this function with the current external datasets.